docs/tutorials/debugging.md

```diff
@@ -0,0 +1,158 @@
+# Debugging 101
+
+This tutorial introduces you to a set of built-in JAX debugging methods — `jax.debug.print`, `jax.debug.breakpoint`, and `jax.debug.callback` — that you can use with various JAX transformations.
+
+Let's begin with `jax.debug.print`.
+
+## JAX `debug.print` for high-level debugging
+
+Python's built-in `print` function can work with some JAX transformations, such as `jax.grad` and `jax.vmap`.
```

Comment:
not totally true... `print` will print the tracer, which even for `grad` and `vmap` is not always what the user expects.